cam/utils.go

```diff
@@ -78,8 +78,9 @@ func lookupCodecTypeByID(codecID C.enum_AVCodecID) codecType {
 
 // ffmpegError returns a string representation of the ffmpeg error code.
 func ffmpegError(ret C.int) string {
-	var errbuf [256]C.char
-	C.av_strerror(ret, &errbuf[0], 256)
+	const errbufSize = 256
```

Comment:
Ah one nit - these never change so can allocate outside the function call - next pr!